### PR TITLE
WIP: Added directive wiring

### PR DESCRIPTION
**Pull request by cliedeman**
Requires  https://github.com/graphql-java-kickstart/graphql-java-tools/pull/312

### CODE REVIEW

File: graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLJavaToolsAutoConfiguration.java [hunk 1]
```diff
@@ -93,6 +96,10 @@ public SchemaParser schemaParser(
             directives.forEach(it -> builder.directive(it.getName(), it.getDirective()));
         }
 
+        if (directingWirings != null) {
```
Comment (rjcruz) [inline graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLJavaToolsAutoConfiguration.java]:
```suggestion
        if (directiveWirings != null) {
```

File: graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLJavaToolsAutoConfiguration.java [hunk 2]
```diff
@@ -93,6 +96,10 @@ public SchemaParser schemaParser(
...
+            directingWirings.forEach(it -> builder.directiveWiring(it));
```
Comment (rjcruz) [inline graphql-spring-boot-autoconfigure/src/main/java/com/oembedler/moon/graphql/boot/GraphQLJavaToolsAutoConfiguration.java]:
```suggestion
            directiveWirings.forEach(it -> builder.directiveWiring(it));
```

**Comment by oliemansm** [comment]
Thanks, it's part of 5.11.0.

**Comment by cliedeman** [comment]
Thanks a lot

**Comment by vinterdo** [comment]
Any example usage? To be precise, I'm trying to use that to wire graphql-java-extended-validation into my spring boot application.

**Comment by sumitbadaya27** [comment]
@vinterdo  did it work for you? i need help with the same please.

**Pull request closed without merging**